index.html

```diff
@@ -0,0 +1,202 @@
+    </li>
+    <li>
+        <a href="https://medium.com/">
+            <img src="img/m.svg" alt="Medium">
+        </a>
+    </li>
+    <li>
+        <a href="https://github.com/djo1975">
+            <img src="img/cat.svg" alt="Github">
+        </a>
+    </li>
+    <li>
+       <a href="https://github.com/djo1975">
+            <img src="img/v.svg" alt="Victory">
+        </a>
+    </li>
+    </ul>
+
+            <div class="container">
+              <div class="items">
+                <div class="portofolio-img"></div>
+                <h2 class="project-title">Tonic</h2>
+                  <div class="c-b-2015">
+                  <h6>CANOPY</h6>
+                  <ul class="c-b-2015-ul">
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                  
+                </div>
+      <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                
+                <ul class="prog-lang">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javaScript</li>
+                </ul>
+                <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+              </div>
+                <div class="items">
+                  <div class="portofolio-img-2"></div>
+                  <h2 class="project-title">Multi-Post <br> Stories</h2>
+                    <div class="c-b-2015">
+                    <h6>CANOPY</h6>
+                    <ul class="c-b-2015-ul">
+                      <li>Back End Dev</li>
+                      <li>2015</li>
+                    </ul>
+                  </div>
+        <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+        
+                  <ul class="prog-lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                  </ul>
+                  <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+                </div>
+
+                <div class="items">
+                    <div class="portofolio-img-3"></div>
+                    <h2 class="project-title">Tonic</h2>
+                      <div class="c-b-2015">
+                      <h6>CANOPY</h6>
+                      <ul class="c-b-2015-ul">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                      </ul>
+                    </div>
+          
+                    <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                    <ul class="prog-lang">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javaScript</li>
+                    </ul>
+                    <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>

```

Comment:
[OPTIONAL]
- [ ] It would be better to use a button here instead of an anchor tag